frontend/projects/upgrade/src/app/features/dashboard/home/components/factorial-experiment-design/conditions-table/conditions-table.component.ts

```diff
@@ -78,7 +78,7 @@ export class ConditionsTableComponent implements OnInit, OnDestroy {
       const formControls = this._formBuilder.group({
         levels: [tableDataRow.levels],
         alias: [tableDataRow.alias],
-        weight: [this.experimentDesignStepperService.formatDisplayWeight(tableDataRow.weight)],
+        weight: [this.experimentDesignStepperService.formatDisplayWeight(tableDataRow.weight),Validators.min(0)],
```

Comment:
If we're not hooking into Angular form validation techniques for showing the message, this is isn't adding anything to what is already there, it can be taken out.